packages/cli/src/decorators/Route.ts

```diff
@@ -13,10 +17,13 @@ const RouteFactory =
 		const controllerClass = target.constructor;
 		const routes = (Reflect.getMetadata(CONTROLLER_ROUTES, controllerClass) ??
 			[]) as RouteMetadata[];
+		const middlewares = options.middlewares ?? [];
+		if (options.throttle && !inTest) middlewares.unshift(rateLimit(options.throttle));
```

Comment:
do we have any other endpoints where we'd like to add throttling as well? if not, I think we should keep this explicitly in `PasswordResetController` for now, until we a need for this elsewhere.

Reply:
Wanted to avoid that because it looks very ugly, but makes sense! Will reverse the last commit